lib/src/Pos/Launcher/Resource.hs

```diff
@@ -221,20 +221,22 @@ getRealLoggerConfig :: MonadIO m => LoggingParams -> m LoggerConfig
 getRealLoggerConfig LoggingParams{..} = do
     let cfgBuilder = productionB
                   <> showTidB
-                  <> maybe mempty prefixB lpHandlerPrefix
+                  <> maybe mempty logsDirB lpHandlerPrefix
```

Comment:
`maybe mempty logsDirB lpHandlerPrefix` can be written as `maybeLogsDirB lpHandlerPrefix`. I introduced this `maybe`-builder just because it wasn't so convenient to use `maybe` for `Maybe` fields...